src/main/webapp/app/pages/curation/header/GeneHeader.tsx

```diff
@@ -145,7 +148,7 @@ const GeneHeader = ({
                 <span className="ms-2">
                   <span className="fw-bold me-2">External Links:</span>
                   <WithSeparator separator={InlineDivider}>
-                    <a href={getCbioportalResultsPageMutationTabUrl(hugoSymbol)} target="_blank" rel="noopener noreferrer">
+                    <a href={getCbioportalResultsPageMutationTabUrl(hugoSymbol ?? '')} target="_blank" rel="noopener noreferrer">
```

Comment:
This default case may be confusing (`''` will just link to current page). Could we disable the link instead?

Reply:
Anchor tags don't have a disabled attribute so I'm going to display a button if there isn't a Hugo Symbol and disable it.

Reply:
changed my mind  
I used styling to disable it